manifests/kustomize/env/namespaced-installation/params.env

```diff
@@ -0,0 +1,20 @@
+appName=pipeline
+appVersion=1.1.0-alpha.1
```

Comment:
Can you update the closest release.sh script to also bump this version on release?

Reply:
sounds good. I will do that.